app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -93,6 +93,10 @@ class PatientRepository @Inject constructor(
         .doOnSubscribe { timingTracker.start(fetchPatientNameAnalytics) }
   }
 
+  fun searchByPhoneNumber(phoneNumber: String): Observable<List<PatientSearchResult>> {
+    TODO("not implemented")
```

Comment:
This code path wont get executed in production because the source event never gets emitted from the UI. Will be implemented in another PR.